common/zip/rules.bzl

```diff
@@ -16,6 +16,7 @@
 #
 
 load("@rules_pkg//:pkg.bzl", "pkg_tar")
+load("@vaticle_bazel_distribution//common/targz:rules.bzl", "assemble_targz")
 load("@vaticle_bazel_distribution//common/tgz2zip:rules.bzl", "tgz2zip")
 
 def _assemble_archive_prefix_file_impl(ctx):
```

Comment:
can we remove `assemble_zip_prefix_file` then?

Reply:
what other things can be cleaned up?

Reply:
@lolski yup, it's no longer needed; addressed in https://github.com/vaticle/bazel-distribution/pull/309/commits/b717657fd6f7f52e5a7dace6c0ef92e5cf637a72

Reply:
@lolski `pkg_tar` is no longer used as well; addressed in https://github.com/vaticle/bazel-distribution/pull/309/commits/ec84bf3ce877608456d8e394ee070143f9edbc85